android/src/main/java/com/apparence/camerawesome/CamerawesomePlugin.java

```diff
@@ -236,10 +228,6 @@ private void _handleCheckPermissions(MethodCall call, Result result) {
 
     }
 
-    private void _handleRequestPermissions(MethodCall call, Result result) {
-        cameraPermissions.checkAndRequestPermissions(pluginActivity);
-    }
-
     private void _handleSetup(MethodCall call, Result result) {
         if (!this.cameraPermissions.hasPermissionGranted()) {
             result.error("MISSING_PERMISSION", "you got to accept all permissions before setup", "");
```

Comment:
Do we need to remove this code, since permissions aren't handled in this lib anymore?

Reply:
The actually permission check is still in this lib but not the request for permissions, I'll update the text a bit.